EStimLibrary.sln

```diff
@@ -7,6 +7,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{7E35BD38-108
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "EStimLibrary", "src\EStimLibrary\EStimLibrary.csproj", "{23710686-EDC8-41FC-AAC8-52B002F488D6}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{6EED0CF2-4443-42D5-8562-2DF5D5697266}"
```

Comment:
Ah, undo changes to the EStimLibrary.sln file. The tests/EStimLibrary.UnitTests/EStimLibrary.UnitTests.sln file should already open both the EStimLibrary source code project and the unit test project